src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.KeyboardHelper.cs

```diff
@@ -118,7 +120,7 @@ public static class KeyboardHelper
 			};
 
 
-			public static void PressKeySequence(string keys, UIElement element = null)
+			public static async void PressKeySequence(string keys, UIElement element = null)
```

Comment:
Should we dispatch the whole method on the dispatcher? That way we won't need to handle exceptions that may push the app to fail when raised on the thread pool.

Reply:
I don't want to change the signature to stay compatible with WinUI tests, which have the same naming. They are also not awaiting, but they do expect the event to be raised asynchronously